docs/guides/expectations/contributing/how-to-contribute-a-new-expectation-to-great-expectations.md

```diff
@@ -223,45 +249,129 @@ The Expectation in the template (expect_column_values_to_equal_three) did not ne
+`value_keys` work for Metrics like `success_keys` do for Expectations, but they are used to determine the value of the metric (hence the name!). If your metric needs additional user-supplied parameters, you add them to the value_keys.
 
 For a map Metric producing a yes/no question answer, you use condition_value_keys (because it’s the condition part of the metric).
 
-Add named arguments to the methods that compute the Metric for each backend in your Metric Provider class:
+* Add named arguments to the methods that compute the Metric for each backend in your Metric Provider class:
+
+````console
 @column_condition_partial(engine=PandasExecutionEngine)
 def _pandas(cls, column, integer=None, **kwargs):
     return column == integer
-Add the new arguments to the test cases in the examples.
-Note:
+````
+
+* Add the new arguments to the test cases in the examples.
+:::
+
+:::note
 
 Some Column Map Metrics that map every row of a column to yes/no need a numeric value pre-computed for each row in order to produce the answer.
 
-This requires defining a new Metric. The parent class of your Metric Provider class (ColumnMapMetricProvider) provides support for this case.
+This requires defining a new Metric. The parent class of your Metric Provider class (`ColumnMapMetricProvider`) provides support for this case.
 
-A good example of this pattern is expect_column_value_z_scores_to_be_less_than - one of the core Expectations.
+A good example of this pattern is `expect_column_value_z_scores_to_be_less_than` - one of the core Expectations.
 
-The Expectation declares “column_values.z_score.under_threshold” as its condition_metric_name (the Metric that answers the yes/no question for every row).
+The Expectation declares “column_values.z_score.under_threshold” as its `condition_metric_name` (the Metric that answers the yes/no question for every row).
 
-The ColumnValuesZScore Metric Provider class that computes this Metric declares an additonal metric:
+The `ColumnValuesZScore` Metric Provider class that computes this Metric declares an additonal metric:
 
+````console
 function_metric_name = "column_values.z_score"
-The class implements methods decorated with @column_function_partial to compute the Z score for every row for each backend.
+````
+
+The class implements methods decorated with `@column_function_partial` to compute the Z score for every row for each backend.
 
 Consult the following files for the details of this pattern:
 
-great_expectations/great_expectations/expectations/core/expect_column_value_z_scores_to_be_less_than.py
-great_expectations/great_expectations/expectations/metrics/column_map_metrics/column_values_z_score.py
-Fill in the ``library_metadata`` dictionary.
+* great_expectations/great_expectations/expectations/core/expect_column_value_z_scores_to_be_less_than.py
```

Comment:
These aren't hyperlinked in the old docs but I think we should do them here. Thoughts?

Reply:
In hindsight, maybe this is best left for the content-focused PRs. Closing this comment down.